interface/patient_file/problem_encounter.php

```diff
@@ -106,12 +107,72 @@
 tr.head   { font-size:10pt; background-color:#cccccc; text-align:center; }
 tr.detail { font-size:10pt; background-color:#eeeeee; }
 </style>
-
+	<?php call_required_libraries(array("jquery-min-3-1-1","font-awesome" , "iziModalToast")); ?>
 <script type="text/javascript" src="../../library/topdialog.js"></script>
 <script type="text/javascript" src="../../library/dialog.js"></script>
 
+
 <script language="JavaScript">
 
+    $(document).ready(function () {
```

Comment:
Is the the same as in common.php above?  If so can we include it instead of copying it?

Reply:
@muarachmann if this code is common to multiple files make a function and put it in a /library/modal_functions.php and just call the function.